momepy/utils.py

```diff
@@ -18,6 +18,57 @@
 ]
 
 
+def deprecated(new_way):
+    """
+    Decorator to mark classes as deprecated and point towards functional API.
+    """
+
+    def decorator(func1):
+        import functools
+
+        @functools.wraps(func1)
+        def new_func1(*args, **kwargs):
+            warnings.warn(
+                f"Class based API like `momepy.{func1.__name__}` is deprecated. "
+                f"Replace it with `momepy.{new_way}` to use functional API instead "
+                "or pin momepy version <1.0. Class-based API will be removed in 1.0. "
+                # "See details at https://docs.momepy.org/en/stable/migration.html",
```

Comment:
Should this line be uncommented? (Also L60 below)?

Reply:
No, as we don't have the migration guide written yet :)